src/client/voice/VoiceConnection.js

```diff
@@ -117,6 +117,15 @@ class VoiceConnection extends EventEmitter {
   }
 
   /**
+   * The current dispatcher
+   * @type {?StreamDispatcher}
+   * @readonly
+   */
+  get dispatcher() {
+    return this.player.streams.size > 0 ? this.player.streams.last().dispatcher || null : null;
```

Comment:
last one as to get latest dispatcher there existing but will change code to be more clear.
Some users hate going all the way to getting dispatcher over voiceConnection so i made shortcut.